submissions/kazeemErinfolami/htmlForms/signUp.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```

Comment:
You're not meant to create this page for now. You need to remove it from this pull request @kazeemerinfolami